package.json

```diff
@@ -7,7 +7,7 @@
     "test": "jest",
     "snyk": "snyk test",
     "clean": "rimraf ./dist && mkdir -p ./dist ./dist/js ./dist/css ./dist/fonts",
-    "start": "./server/server.js",
+    "start": "nodemon ./server/server.js",
```

Comment:
Does the nodemon need to be here? Shouldn't we just change the `watch` command to `npm run watch:server` instead of `npm start`? Also, you can change the `watch:server` command's `-x` flag to `'npm start'` (which runs the special script named start ^_~.. equivalent but slightly shorter)